utils/oscap-cvrf.c

```diff
@@ -151,7 +151,9 @@ static int app_cvrf_export(const struct oscap_action *action) {
 		if (oscap_err())
 			fprintf(stderr, "%s %s\n", OSCAP_ERR_MSG, oscap_err_desc());
 
-	oscap_source_free(import_source);
+	/* TODO: Refactor, cvrf_index_parse_xml (called by oscap_source_new_from_file) frees its argument as an unexpected side-effect.
+	 * oscap_source_free(import_source);
+	 */
```

Comment:
Maybe intention in https://github.com/OpenSCAP/openscap/blob/2265c50484f76baf91e4420b247e8e04a0c34cf9/src/CVRF/cvrf_priv.c#L2335 was to `free(buffer)`, allocated by `oscap_source_get_raw_memory()`?

I don't understand where `buffer`  is used in [`cvrf_index_parse_xml`](https://github.com/OpenSCAP/openscap/blob/2265c50484f76baf91e4420b247e8e04a0c34cf9/src/CVRF/cvrf_priv.c#L2323).
Maybe to parse content put in `buffer`? Where does parsing occur?